core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/ldaps/LdapConfigV4Endpoint.java

```diff
@@ -61,6 +63,13 @@ LdapV4Responses list(@PathParam("workspaceId") Long workspaceId, @QueryParam("en
             nickname = "deleteLdapConfigsInWorkspace")
     LdapV4Response delete(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String ldapConfigName);
 
+    @DELETE
+    @Path("")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = LdapConfigOpDescription.DELETE_MULTIPLE_BY_NAME_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.LDAP_CONFIG_NOTES,
+            nickname = "deleteMultipleLdapConfigsInWorkspace")
```

Comment:
Should just be "deleteLdapConfigsInWorkspace" to match the other nicknames

Reply:
The problem there is that the singular endpoint is already called "deleteLdapConfigsInWorkspace." Any suggestions?

Reply:
Change the singular endpoint to "deleteLdapConfigInWorkspace"?

Reply:
Making the change to the singular as well as the multiple here.